### ISSUE TITLE
New Example: Create table with inputs in cells

### ISSUE THREAD

**Issue by aguirrel** [formly-js/angular-formly-website#67]
I would like an example that demonstrates: how to create a table with inputs in cells like
![lineitem](https://cloud.githubusercontent.com/assets/697379/9136283/cade56aa-3ceb-11e5-93f6-122376c58b07.png)

Is it possible?

Thanks a lot!

**Comment by kentcdodds** [comment]
This is definitely possible, but I don't think that I have the time to create an example like this for you. You could give it a shot and follow the instructions here: http://help.angular-formly.com

**Comment by aguirrel** [comment]
Hi @kentcdodds, thanks! Could you give me some little help how to structure it? Should I use Wrappers? Only templates? Anything you say will be helpful :)

**Comment by kentcdodds** [comment]
So, you'll either want to have a formly-form for each row or use a field group. I would make each item in the row a field. It sort of depends on your use case, but they probably don't need wrappers. Only use wrappers if you find yourself repeating the same html in multiple templates. You might also consider creating a template manipulator.

**Comment by aguirrel** [comment]
Thanks a lot! I will come back with an example.

**Comment by aguirrel** [comment]
Hi @kentcdodds I did and example at http://plnkr.co/edit/LtDqUdT47kQgKsqMQHgq?p=preview

Main problem is that fields code and description are on the same <TD> tag, I can't manage to do td tag individual for each subfield.  Could you have a minute to give me some advice?

Thanks a lot!

Best regards.

**Comment by kentcdodds** [comment]
Does this help? http://plnkr.co/edit/g9zFSylI2BB2oBBA0lmc?p=preview

**Comment by aguirrel** [comment]
Hi @kentcdodds! Thanks a lot for your time!  Is much near what I need, but, I'm wondering, why 'TR's tags are outside TABLE tag?

**Comment by kentcdodds** [comment]
Yeah, that's weird... I'm not sure why that's happening... Sorry, I don't have time to diagnose it right now :-/

**Comment by aguirrel** [comment]
NP, anyway you gave me some clue about the solution. Thanks! and great job!

**Comment by aguirrel** [comment]
Hi @kentcdodds, I found a solution for this behaviour, but I'm a step behind my objetive.

http://plnkr.co/edit/XhBBkBybT3SLHkvFp8NZ?p=preview

Now problem is I can't create tbody.  I sould have thead and tbody, if you check example and changes in HTML:

`<formly-form model="vm.model" root-el="div" field-root-el="table" fields="vm.fields" options="vm.options" form="vm.form">`

to 

`<formly-form model="vm.model" root-el="table" field-root-el="tbody" fields="vm.fields" options="vm.options" form="vm.form">`

I just get an error: Error: [$compile:tplrt] Template for directive 'formlyForm' must have exactly one root element.

Thanks a lot!

**Comment by kentcdodds** [comment]
I'm not seeing the issue you're describing, for me it just doesn't output
at all...

P.S. I hate tables :-( Is there any way you could use bootstrap columns or
flexbox?

-Kent C. Dodds

**Comment by aguirrel** [comment]
I will try with flexbox.  I like table becouse is more standard ;)  Thanks a lot!!!

**Comment by mikehaas763** [comment]
@aguirrel Did you ever come to a solution for this?

**Comment by aguirrel** [comment]
@mikehaas763, no, I couldn't create tbody and thead for table.  This directive is awesome, but DOM is polluted so much.  I ended up using https://github.com/json-schema-form/angular-schema-form which is great and easier to extend, I did this table template and 2 more in some hours.

**Comment by EsaudTapia** [comment]
Did you manage to make the table?
